packages/graphql-hooks/index.d.ts

```diff
@@ -183,10 +187,16 @@ export interface UseClientRequestOptions<ResponseData = any, Variables = object>
   client?: GraphQLClient
 }
 
+type RefetchAferMutationsData = {
+  mutation: string
+  filter?: (variables: any) => void
+}
+
 export interface UseQueryOptions<ResponseData = any, Variables = object>
   extends UseClientRequestOptions<ResponseData, Variables> {
   ssr?: boolean
   skip?: boolean
+  refetchAfterMutations?: RefetchAferMutationsData
```

Comment:
Is this type signature correct? The usage seems to suggest that it's an array.

Reply:
Yeah, it's incorrect. Thanks, I'll fix it.